lib/linguist/languages.yml

```diff
@@ -4110,6 +4110,8 @@ Shell:
   - ".command"
   - ".fcgi"
   - ".ksh"
+  - ".mksh"
+  - ".pdksh"
```

Comment:
* 13 mksh https://github.com/search?utf8=%E2%9C%93&q=extension%3Amksh+mksh&type=
* 14 pdksh https://github.com/search?utf8=%E2%9C%93&q=extension%3Apdksh+pdksh&type=

Indeed, it's pretty rare. Removed.